testing/attica-qt5/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
+pkgname=attica-qt5
```

Comment:
Why does this package name divert from upstream package name?

Do you honestly believe it will be built against Qt4?